requirements/common.txt

```diff
@@ -0,0 +1 @@
+requests==2.22.0
```

Comment:
ok kinda related to https://github.com/CedricCabessa/coffeeconnection/pull/4#discussion_r341074667
I read
https://pip.pypa.io/en/stable/user_guide/#requirements-files
https://caremad.io/posts/2013/07/setup-vs-requirement/

We don't use requirement.txt file at $JOB so I'm not really aware of best practices.

May be we can feed it with `pip freeze` (from a fresh venv) and keep the "not pinned" in setup.py
(so we add appdir==xxx) here.

(I can do that later, it is not totally related to this PR)
That's why I have tryied to stay away for requierement.txt :-)